tracing/README.md

```diff
@@ -0,0 +1,37 @@
+# Tracing package
```

Comment:
We decided to go H1s = Title Case, H2s = Sentence case, so this should be "Tracing Package" instead.